pandas/core/missing.py

```diff
@@ -92,8 +94,7 @@ def clean_fill_method(method, allow_nearest=False):
     return method
 
 
-def clean_interp_method(method, **kwargs):
-    order = kwargs.get("order")
+def clean_interp_method(method: str, order: Optional[int] = None, **kwargs) -> str:
```

Comment:
Are kwargs even required here any more?

Reply:
I would imagine so, split out order so that can add type annotation for internal checks. will double check though.

Reply:
it doesn't appear that we pass the kwargs through to np.interp (left, right and period) but do for scipy.interpolate.interp1d

did you mean kwargs not used in this function? That's true, but would need to do `order = kwargs.get("order")` in the calling functions if I remove kwargs here. I don't see that as an improvement.

should I revert this change?